### PR TITLE
Adds a configuration dialog for setting the direnv path

### PR DESCRIPTION
**Pull request by Fapiko**
I could not get this plugin working locally because direnv was installed with nvm and IntelliJ wasn't finding it. This adds a config to set the direnv path. If the path is set and not empty it will use that when executing the direnv call. This should be fully backwards compatible.

Sorry that it's in Java - haven't done any Kotlin (nor much Java lately) but really want this plugin to actually work so I don't have to hack direnv support into my run configs anymore.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -10,7 +10,7 @@ plugins {
     // Kotlin support
     id("org.jetbrains.kotlin.jvm") version "1.5.10"
     // gradle-intellij-plugin - read more: https://github.com/JetBrains/gradle-intellij-plugin
-    id("org.jetbrains.intellij") version "1.0"
+    id("org.jetbrains.intellij") version "1.6.0"
```
Comment (Fapiko) [inline build.gradle.kts:13]: This made it work with the latest version of GoLand

File: src/main/kotlin/systems/fehn/intellijdirenv/services/DirenvProjectService.kt [hunk 2]
```diff
@@ -69,7 +70,6 @@ class DirenvProjectService(private val project: Project) {
                         MyBundle.message("exceptionNotification"),
                         e.localizedMessage,
                         NotificationType.ERROR,
-                        null,
```
Comment (Fapiko) [inline src/main/kotlin/systems/fehn/intellijdirenv/services/DirenvProjectService.kt:72]: Fixed a deprecation warning

**Comment by fehnomenal** [comment]
Hey, thanks for contributing. I always wanted to add a settings dialog like this :+1:

When the build finishes I will do some quick tests.

**Comment by fehnomenal** [comment]
Ah, could you please add a chengelog entry below the [Unreleased] section?

**Pull request merged**